addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -917,7 +917,7 @@ registry.anchorSlide = publicWidget.Widget.extend({
      * @private
      */
     _onAnimateClick: function (ev) {
-        if (this.$target[0].pathname !== window.location.pathname) {
+        if (this.$target[0].pathname.replaceAll("/", "") !== window.location.pathname.replaceAll("/", "")) {
```

Comment:
About this commit, I don't get it, can you explain a bit?

How is this ever not gonna be true when in alternate language?

It seems like if you are on a translated version of the website, the link in the dom is the original link without the lang, so it won't ever be equal to the current url?

See this video https://watch.screencastify.com/v/PCW118kCEnBQvE7qLodJ

Link pathname is `/` but the current page pathname is `/fr/` ?
Same for other URLs
![image](https://user-images.githubusercontent.com/30048408/159518064-03d0f448-d778-4d6c-a71d-21232b83bb74.png)

Reply:
I tested and It works on my side, `window.location.pathname` and `this.$target[0].pathname` are indentical on fr_BE pages. There must be something different in our way of testing. Maybe we can see this together.

![FireShot Capture 1184 - Capture d’écran de 2022-03-24 14-04-55 png (3840×1080) - ](https://user-images.githubusercontent.com/52911687/159922602-21c8a8fe-6cc3-406d-ad0c-9cb6a12de926.jpg)

Reply:
I am not doing anything special really, here is a video: https://watch.screencastify.com/v/sfcTA7FX1ENfG36mRrDu
I just:
- Picked your latest runbot (you will see the / and /contactus pages have links I made)
- Created links here and there through CTRL+K / link dialog
- Saved
- Went to french version
- Clicked on links

Reply:
I cant reproduce the bug, I tested a lot of time (locally and on the same runbot as you) and each time URL's in the DOM are correctly updated with the current lang

Reply:
Ok I was able to reproduce the bug. This only happens when debug mode is enabled.
This is because when debug mode is enabled, link URLs do not have the lang. 
It also seems that the lang ​​are not in the links URLs if debug mode is disabled (but only if it was enabled before).
It is related to the following line in `def _post_processing_att()`
```
        if options.get('inherit_branding') or options.get('rendering_bundle') or \
           options.get('edit_translations') or options.get('debug') or (request and request.session.debug):
            return atts
```
This is therefore not really linked to the code of this PR, @rdeodoo  I let you take a look at it to see if something needs to be done in a new PR

Reply:
Definitely something weird here. Weirdest part was when one out of every 2 page refresh was showing the lang prefix, but the other refresh not.
I am not 100% convinced by the debug idea, but it could very well be that. Might also be some website.page caching issue.
I'll investigate when I'll be reviewing this PR again why is this happening and if a fix is needed.

Maybe keep this review unresolved for now in case someone faces the same issue.

Reply:
Since this misbehavior is only related to debug mode because it bypass some mechanism related to URL generation, I'd not fix it. It's ok if the scroll is not working smoothly (does a reload instead) in debug. There is many flows and features that won't work as expected in debug.